src/table/utils/styling-utils.ts

```diff
@@ -6,6 +6,13 @@ import { GeneratedStyling, CellStyle, FooterStyle } from '../types';
 import { SelectionStates, PAGINATION_HEIGHT } from '../constants';
 import { SELECTION_STYLING, COLORING } from '../styling-defaults';
 
+const LINE_HEIGHT = 4 / 3;
+const CELL_PADDING_HEIGHT = 8;
+const CELL_BORDER_HEIGHT = 1;
```

Comment:
these would fit nicely in styling-defaults. Possibly that the file should have a different name

Reply:
I would think they fitter better in a "constants" file, but since they are only used here I don't think it will add much value to move them

Reply:
sure, they can be here since until they are needed elsewhere